examples/latent_sde.py

```diff
@@ -71,24 +71,47 @@ def f(self, t, y):  # Approximate posterior drift.
         inp = torch.cat((torch.sin(t), torch.cos(t), y), dim=-1)
         return self.net(inp)
 
+    def f_aug(self, t, y):
+        y = y[:, 0:1]
+        f, g, h = self.f(t, y), self.g(t, y), self.h(t, y)
+        z = f - h
+        g = torch.where(
```

Comment:
Could we abstract this and the next line into a helper function `_stable_division` as before in `misc`? 

The functionality of this step is clearer if we give these lines of code a name by wrapping them in a help function. 

Minor nit: You may also call the resulting value `u` to better match the derivations.

Reply:
Done!